sql/000001.sql

```diff
@@ -79,7 +79,7 @@ create trigger _900_notify_worker after insert on :GRAPHILE_WORKER_SCHEMA.jobs f
 create function :GRAPHILE_WORKER_SCHEMA.add_job(
   identifier text,
   payload json = '{}',
-  queue_name text = public.gen_random_uuid()::text,
+  queue_name text = 'PLACEHOLDER',
```

Comment:
This feels weird, but technically at this point in time the column is non-nullable so I used a string that makes it clear that it isn't actually going to be used.

Reply:
Use `null /* was gen_random_uuid(), but later removed dependency */`